spec/models/ability_spec.rb

```diff
@@ -210,7 +208,7 @@
       it{ should_not be_able_to(:manage, my_conference.tickets.first) }
       it{ should_not be_able_to(:manage, conference_public.tickets.first) }
 
-      it{ should_not be_able_to(:manage, my_registration) }
+      it{ should be_able_to(:manage, my_registration) }
```

Comment:
@differentreality user with cfp role should be able to manage his own registration (my_registration)?